feature-group-definitions/background-gradients.dist.yml

```diff
@@ -0,0 +1,63 @@
+# Generated from: background-gradients.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: Background gradients
+description: The `linear-gradient()`, `radial-gradient()`, and `conic-gradient()` CSS functions create backgrounds that progress between two or more colors.
+spec: https://drafts.csswg.org/css-images-4/#gradients
+caniuse:
+  - css-gradients
+  - css-repeating-gradients
+  - css-conic-gradients
+status:
+  baseline: high
+  baseline_low_date: 2020-11-17
```

Comment:
A lot of this was supported even earlier:
https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/linear-gradient#browser_compatibility

I'll back this out to consider separately.